doc/lua_api.txt

```diff
@@ -2388,8 +2401,85 @@ offsets when migrating:
 | list    |            |         | Spacing is now 0.25 for both directions, meaning lists will be taller in height
 | label   | 0, +0.3    |         | The first line of text is now positioned centered exactly at the position specified
 
+Styling Formspecs
+-----------------
 
-
+Formspec elements can be themed using the style elements:
+
+    style[<name>;<prop1>;<prop2>;...]
+    style_type[<type>;<prop1>;<prop2>;...]
```

Comment:
Interpreted as HTML tags. Escape the entire line using code notation.

Reply:
It's already in a code block